app/src/services/subscription.service.js

```diff
@@ -30,6 +30,10 @@ class SubscriptionsService {
         // Build subscription params
         body.params = {};
 
+        if (area.geostoreDataApi) {
```

Comment:
if both geostoreDataApi  and geostore exist, this will lead to problems. I know that, by definition, they should be mutually exclusive, but if we rely on that, we should make it explicit - aka throw an error if both are set on create/update. Until then, having this sort of "half enforced assumption" will lead to confusing behavior.

Reply:
Agreed, will do that :+1: